### PR TITLE
Update python server

### PR DESCRIPTION
**Pull request by PLoic**

### CODE REVIEW

**Comment by PLoic** [comment]
@quentinus95  Can you check the code and test if it working well for you ? Thanks

File: server.py [hunk 1]
```diff
@@ -25,81 +27,73 @@ def do_AUTHHEAD(self):
...
-def test(HandlerClass = AuthHandler,
-         ServerClass = BaseHTTPServer.HTTPServer):
-    BaseHTTPServer.test(HandlerClass, ServerClass)
-
 
 @contextmanager
-def pushd(newDir):
-    previousDir = os.getcwd()
-    os.chdir(newDir)
+def pushd(new_dir):
```
Comment (quentinus95) [inline server.py:47]: Could you add a simple comment after this line to briefly explain why it is needed ?

Comment (PLoic) [reply server.py:47]: Done, need an approbation

Comment (quentinus95) [reply server.py:47]: k

**Review by quentinus95 — APPROVED** [review]

**Pull request merged**